packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -189,10 +189,26 @@ export const FilterList: React.FunctionComponent = () => {
 export const Table: React.FunctionComponent = () => {
   const [pageSize, setPageSize] = useState(10)
```

Comment:
The default value of page size should come from `resource?.perPageOptions?.[0]`.